orchagent/orch.h

```diff
@@ -243,6 +244,8 @@ class Orch
     virtual task_process_status handleSaiRemoveStatus(sai_api_t api, sai_status_t status, void *context = nullptr);
     virtual task_process_status handleSaiGetStatus(sai_api_t api, sai_status_t status, void *context = nullptr);
     bool parseHandleSaiStatusFailure(task_process_status status);
+
+    ResponsePublisher m_publisher{"APPL_STATE_DB"};
```

Comment:
"APPL_STATE_DB" is the DB name, not the table name.
The table name is passed in the publish() API.
In implementation, one publisher can write to multiple tables. We do it in this way instead of one publisher per table. The main purpose for this is to use a single instance of publisher to write to all tables instead of creating multiple publishers.

Reply:
I meant DB name

Reply:
Addressed